### ISSUE TITLE
Returning 500 Error During Requests For Automat KP's

### ISSUE THREAD

**Issue by uhbrar** [TranslatorSRI/Plater#39]
Automat KP's will return 500 errors when making a POST request at the /query endpoint. This happens both when going through Strider and when calling the API directly. Only happens with certain data values, but various requests will cause this to happen. The attached file shows some examples of requests and data values that cause this error to be returned.

[response_1625760453713.txt](https://github.com/TranslatorSRI/Plater/files/6926522/response_1625760453713.txt)

**Comment by YaphetKG** [comment]
Thanks for opening this issue @uhbrar, 

It seems like Plater is hitting an error when trying to find inverse of some predicates :

```
{
  "message": {
    "query_graph": {
      "nodes": {
        "n0": {
          "categories": [
            "biolink:ChemicalSubstance"
          ],
          "is_set": false
        },
        "n1": {
          "ids": [
            "HGNC:3236"
          ],
          "categories": [
            "biolink:Gene"
          ],
          "is_set": false
        }
      },
      "edges": {
        "e0": {
          "subject": "n0",
          "object": "n1",
          "predicates": [
            "biolink:decreases_abundance_of",
            "biolink:decreases_activity_of",
            "biolink:decreases_expression_of",
            "biolink:decreases_synthesis_of", 
            "biolink:increases_degradation_of", 
            "biolink:disrupts",
            "biolink:negatively_regulates_entity_to_entity"
          ]
        }
      }
    }
  }
}
```

This is one of the questions sent by strider from the file above. And the predicate "biolink:negatively_regulates_entity_to_entity" seems to be the one that causing the 500. 
Specific error from logs: 
```
....
    cypher = self.compile_cypher()
  File "/home/plater/Plater/PLATER/services/util/question.py", line 46, in compile_cypher
    return get_query(self._question_json[Question.QUERY_GRAPH_KEY])
  File "/usr/local/lib/python3.8/site-packages/reasoner/cypher.py", line 182, in get_query
    query = transpile_compound(qgraph, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/reasoner/cypher.py", line 34, in transpile_compound
    return match_query(
  File "/usr/local/lib/python3.8/site-packages/reasoner/matching.py", line 309, in match_query
    clauses.append(match_edge(
  File "/usr/local/lib/python3.8/site-packages/reasoner/matching.py", line 259, in match_edge
    eref = EdgeReference(qedge_id, qedge)
  File "/usr/local/lib/python3.8/site-packages/reasoner/matching.py", line 169, in __init__
    inverse = el.inverse
AttributeError: 'NoneType' object has no attribute 'inverse'
```

**Comment by YaphetKG** [comment]
Issue is resolved with PR #40 .
